Packs/rasterize/ReleaseNotes/1_2_4.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Rasterize
+- Added an option in the command **rasterize** to include the URL bar in the screenshot.
+- Updated the Docker image to: *devdemisto/chromium:1.0.0.32875*.
```

Comment:
don't forget to update this release note too when changing to the production image